site/pages/dashboard.page.tsx

```diff
@@ -1,3 +1,4 @@
+import dayjs from "dayjs";
```

Comment:
We dont want dayjs being imported to pages. We have a file called dates.ts inside the utils where all the date functions live.

Reply:
done, using functions from dates.js where possible